packages/libs/components/src/map/SemanticMarkers.tsx

```diff
@@ -205,10 +208,34 @@ export default function SemanticMarkers({
         consolidatedMarkers.find(({ props }) => id === props.id)
       );
 
-      if (prunedSelectedMarkers.length < selectedMarkers.length)
+      if (prunedSelectedMarkers.length < selectedMarkers.length) {
+        const consolidatedMarkersElementLength =
+          consolidatedMarkers.length === 0
+            ? 0
+            : consolidatedMarkers[0].props.id.length;
+
+        const selectedMarkersElementLength =
+          selectedMarkers.length === 0 ? 0 : selectedMarkers[0].length;
+
+        const message =
+          consolidatedMarkersElementLength !== selectedMarkersElementLength
+            ? 'marker selection has been cancelled because aggregation level has changed due to zooming'
```

Comment:
Consistent capitalization to match previous marker-related snackbars
```suggestion
            ? 'Marker selection has been cancelled because aggregation level has changed due to zooming'
```

Reply:
@bobular Yep I will change it. Thanks 👍